bigframes/core/utils.py

```diff
@@ -84,26 +85,42 @@ def get_standardized_ids(
         Tuple of (standardized_column_ids, standardized_index_ids)
     """
     col_ids = [
-        UNNAMED_COLUMN_ID if col_label is None else str(col_label)
+        UNNAMED_COLUMN_ID if col_label is None else label_to_identifier(col_label)
         for col_label in col_labels
     ]
     idx_ids = [
-        UNNAMED_INDEX_ID if idx_label is None else str(idx_label)
+        UNNAMED_INDEX_ID if idx_label is None else label_to_identifier(idx_label)
         for idx_label in idx_labels
     ]
 
-    ids = idx_ids + col_ids
+    ids = disambiguate_ids(idx_ids + col_ids)
+
+    idx_ids, col_ids = ids[: len(idx_ids)], ids[len(idx_ids) :]
+
+    return col_ids, idx_ids
+
+
+def label_to_identifier(label: typing.Hashable, strict: bool = False) -> str:
```

Comment:
would be nice to have unit tests on these utils methods

Reply:
Since utils are not part of the public API, I would prefer to test them via the public APIs that use them. I tried to include some unusual labels in the test cases.

Reply:
Sounds good, let's add these two kinds of labels to test the strict mode:

1. Leading spaces followed by alpha-numeric
2. Leading spaces followed by `_`